save-backend/src/main/kotlin/com/saveourtool/save/backend/security/OrganizationPermissionEvaluator.kt

```diff
@@ -123,3 +145,6 @@ class OrganizationPermissionEvaluator(
         val contestCreatorMinimalRole = Role.ADMIN
     }
 }
+
```

Comment:
It is not that obvious when the receiver is `OrganizationStatus`. Can status be banned (meaning someone banned the status)?

Reply:
See it is also done